integrations/access/opsgenie/app.go

```diff
@@ -67,6 +68,8 @@ type App struct {
 	opsgenie   *Client
 	mainJob    lib.ServiceJob
 	conf       Config
+
+	accessMonitoringRules *accessmonitoring.AccessMonitoringRuleHandler
```

Comment:
Consider this rename so you don't have to say "access monitoring" twice.
```suggestion
	accessMonitoringRules *accessmonitoring.RuleHandler
```